CoreRPC/Transport/NamedPipe/NamedPipeClientTransport.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.IO.Pipes;
+using System.Threading.Tasks;
+
+namespace CoreRPC.Transport.NamedPipe
+{
+    public sealed class NamedPipeClientTransport : IClientTransport
+    {
+        private readonly string _serverName;
+        private readonly string _pipeName;
+
+        public NamedPipeClientTransport(string pipeName, string serverName = ".")
+        {
+            _serverName = serverName;
+            _pipeName = pipeName;
+        }
+
+        public async Task<byte[]> SendMessageAsync(byte[] message)
+        {
+            var pipe = new NamedPipeClientStream(_serverName, _pipeName, PipeDirection.InOut);
```

Comment:
`using(var pipe = ...`

Reply:
If I try disposing client pipe here, `System.ObjectDisposedException` exception is thrown. Probably this is because pipe is already closed by `NamedPipeServerStream`